pkgs/development/python-modules/pyocr/default.nix

```
@@ -7,12 +7,15 @@
 , isPy3k
 , substituteAll
 , pytestCheckHook
+, setuptools
+, setuptools-scm
 }:
 
 buildPythonPackage rec {
   pname = "pyocr";
-  version = "0.8.3";
+  version = "0.8.5";
   disabled = !isPy3k;
+  format = "pyproject";
```

```suggestion
  pyproject = true;
```
FYI: We have a new parameter for pyproject.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters